File: distance/distance.clj
(ns distance)

(defn euclid [a b]
  (let [[x1 y1] a
        [x2 y2] b
        dx (- x1 x2)
        dy (- y1 y2)]
    (Math/sqrt (+ (* dx dx) (* dy dy)))))

(println (euclid [1 1] [2 2]))

File: distance/distance.py
#!/usr/bin/env python3

import math


def euclid(a, b):
    d = math.sqrt((a[0] - b[0]) ** 2 + (a[1] - b[1]) ** 2)
    print(d)


euclid((1, 1), (2, 2))
